doc/presentations.rst

```diff
@@ -27,6 +27,11 @@ specific subject areas:
 Videos
 ======
 
+- An introduction to scikit-learn `Part
+  I <https://conference.scipy.org/scipy2013/tutorial_detail.php?id=103>`_ and
```

Comment:
^ Think you mean <https://conference.scipy.org/scipy2013/tutorial_detail.php?id=107>. 103 leads me to matplotlib...